src/utils/currency.ts

```diff
@@ -165,6 +176,7 @@ export function getCurrencyIdentifier(currency: CurrencyExt): CurrencyIdentifier
     return { token: currency.ticker };
 }
 
+// TODO: add StandardLPToken and StableLPToken
```

Comment:
Something to consider for the TODOs:
Would it make sense to capture those in one or several issues? Just so we don't lose track with the many small changes.

Reply:
Yes, I already created several issues.